CHANGES.md

```diff
@@ -4,6 +4,8 @@
 
   + Remove the 'let-open' option, deprecated since 0.16.0 (#1563, @gpetiot)
 
+  + Format objects and classes in a compact way (#1569, @bikallem)
```

Comment:
I would say "Format objects and classes consistently with structure and signature items" (as compact only depends on the options), I already renamed the PR